terraform/helm/validator/values.yaml

```diff
@@ -31,6 +31,9 @@ validator:
     sync_only: false
     mempool_poll_count:
     round_initial_timeout_ms:
+    default_prune_window: 10000000
+    state_store_prune_window: 0
```

Comment:
Let's set the default to what we have in code `1000000`,

Reply:
@sitalkedia default is 10M, 7 zeros

Reply:
@sherry-x - That's the default_pruner_window. For state_store_pruner_window, the default value is 1M - https://github.com/aptos-labs/aptos-core/blob/main/config/src/config/storage_config.rs#L101